CMSgov/design-system
# WETGPCII-2788-trailing-decimal-truncation

## Summary
ISSUE: Not able to move forward when APTC is selected little higher with third decimal after PCII truncates the trailing decimal

STEPS TO RECREATE:
    Create APTC eligible FA app and navigate to PCII
    Now, select 'SOME of the tax credit each month' option
    Enter eligible aptc with third decimal {Example from video $1099.008}
    Notice that decimal is truncated and amount adjusts to $1099
    Also, notice the error message saying 'Enter $1,099 or less.' even the amount seen is $1099 after PCII truncated the decimal.

EXPECTATION: No error message to be seen and able to move forward

- [Jira Story WETGPCII-2788](https://jira.cms.gov/browse/WETGPCII-2788)

## How to test

1. Run Unit Tests
2. Start Storybook
    - Under the list of Components, Click "TextField > MaskedCurrency"
    - Follow the Guidelines for testing the Masked TextField Element
      - Specifically, ensure the input elements is checking against the "pattern" attribute regex. No alpha numbers should be allowed
      - Ensure the error handling is correct border is turning red when alpha characters are entered
      - Ensure truncation of numbers consisting of three decimal points takes place
      - Ensure that number with no valid decimal points are converted to whole numbers

## Checklist




### If this is a change to design:


### If this is a change to code:


### If this is a change to documentation:
